### ISSUE TITLE
Some variables declared as enum are not detected

### ISSUE THREAD

**Issue by michaelopdenacker** [bootlin/elixir#250]
For example pin_config_param https://elixir.bootlin.com/linux/v6.4-rc2/C/ident/pin_config_param.
Here it's not an "enum" symbol, just a variable declared as enum.

**Comment by fstachura** [comment]
I'm not sure what this issue refers to, in current index pin_config_param is an enum, not a variable. Maybe that wasn't the case with the corrupted index mentioned in #246? 

From #246 

> v6.4 database is definitely corrupted. On top of that (dunno if it's a separate bug) the enum fields are not indexed. Ex.: https://elixir.bootlin.com/linux/v6.4-rc2/C/ident/pin_config_param.

This is probably #221, fixed in #344 (not merged yet). Notice that all enumerators contain CONFIG_.

**Comment by andy-shev** [comment]
@fstachura 
Look at https://elixir.bootlin.com/linux/v6.13.4/A/ident/PIN_CONFIG_BIAS_DISABLE, for example. It's empty, why it must not.

**Comment by fstachura** [comment]
Yeah, but I think that's #221. Elixir skips all identifiers that match `*CONFIG_*`. It was supposed to be `^CONFIG_`, probably, to skip defines exposed by Kconifg. See [this](https://github.com/bootlin/elixir/blob/master/script.sh#L169). The fix is ready (#344).
